src/components/Practice/Mechanics7.jsx

```
@@ -614,111 +650,116 @@ const Mechanics7 = ({
+                      {elem?.split("")?.map((char, index) => (
+                        <span
+                          style={{
+                            color: wIndex == 0 ? "#1897DE" : "#000000",
+                            fontWeight: 700,
+                            fontSize: "25px",
+                            lineHeight: "87px",
+                            letterSpacing: "2%",
+                            fontFamily: "Quicksand",
+                            marginLeft: index > 0 ? "12px" : undefined,
+                          }}
+                        >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)